examples/lib/stories/components/components.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:dashbook/dashbook.dart';
+import 'package:flame/game.dart';
+
+import '../../commons/commons.dart';
+import 'composability.dart';
+import 'debug.dart';
+
+void addComponentsStories(Dashbook dashbook) {
+  dashbook.storiesOf('Components')
+    ..add(
+      'Composability',
+      (_) => GameWidget(game: Composability()),
```

Comment:
Do we want to create the game inside of the `GameWidget`, don't we want to create it outside and reference it?

Reply:
we can improve the structure after this PR is merged